tests/src/test/scala/akka/kafka/scaladsl/TransactionsSpec.scala

```diff
@@ -321,7 +321,7 @@ class TransactionsSpec extends SpecBase with TestcontainersKafkaLike {
               case (_, value) => duplicates.contains(value)
             }
             .groupBy(_._2) // message
-            .mapValues(_.map(_._1)) // keep offset
+            .map { case (k, v) => (k, v.map(_._1)) } // keep offset
```

Comment:
Why did this change?

Reply:
To workaround Scala collections changes and compatibility across 2.12 and 2.13 builds.  I added a comment.